goss.yaml

```diff
@@ -140,6 +140,10 @@ command:
   /usr/local/bin/docker-low-disk-gc:
     exit-status: 0
 
+  docker buildx build -f tests/Dockerfile --progress=plain -t buildkite-postgres:latest tests:
+    exit-status: 0
+    timeout: 30000
+
```

Comment:
This tests building a problematic container image

Reply:
Worth adding this as a comment in the file?